pkg/cluster/fetch.go

```diff
@@ -112,6 +139,20 @@ func bundlesNamespacedKey(cluster *v1alpha1.Cluster) (name, namespace string) {
 	return name, namespace
 }
 
+func getEKSAReleaseForCluster(ctx context.Context, cluster *v1alpha1.Cluster, fetch EKSAReleaseFetch) (*v1alpha1release.EKSARelease, error) {
+	if cluster.Spec.EksaVersion == nil {
+		return nil, fmt.Errorf("Cluster's EksaVersion is nil")
+	}
+	version := string(*cluster.Spec.EksaVersion)
+	name := v1alpha1release.GenerateEKSAReleaseName(version)
+	eksaRelease, err := fetch(ctx, name, constants.EksaSystemNamespace)
+	if err != nil {
+		return nil, fmt.Errorf("fetching EKSARelease for cluster: %v", err)
```

Comment:
Do we have a different EKSARelease and EksaRelease still?

Reply:
yes